src/templates/About/index.tsx

```diff
@@ -5,10 +5,13 @@ import { withRouteData } from 'react-static';
 import { ActionBar, IActionBar } from '../../components/ActionBar';
 import { Businesses, IBusinesses } from '../../components/Businesses';
 import { Collage, ICollage } from '../../components/Collage';
+import { FeatureSection, FeatureStrip, IFeatureSection } from '../../components/FeatureSection';
 import { Hero, IHero } from '../../components/Hero';
 import { Image } from '../../components/Image';
 import { Layout } from '../../components/Layout';
 import { IPressSection, PressSection } from '../../components/PressSection';
+import { Section } from '../../components/Section';
+// import { IValueSection, ValueSection } from '../../components/Value';
```

Comment:
I suggest keeping this uncommented and just removing the values section from the YAML file

```suggestion
import { IValueSection, ValueSection } from '../../components/Value';
```